Avoid 
Undefined                       first referenced
 symbol                             in file
icu_55::TimeZone::recreateDefault()    Unified_cpp_js_src8.o  (symbol scope specifies local binding)
--- firefox-43.0b3/intl/icu-patches/bug-1172609-timezone-recreateDefault.diff.~1~	2015-11-13 01:48:42.000000000 +0300
+++ firefox-43.0b3/intl/icu-patches/bug-1172609-timezone-recreateDefault.diff	2015-11-19 11:37:48.996563297 +0300
@@ -37,7 +37,7 @@
      return (DEFAULT_ZONE != NULL) ? DEFAULT_ZONE->clone() : NULL;
  }
  
-+void
++void U_EXPORT
 +TimeZone::recreateDefault()
 +{
 +    TimeZone *default_zone = TimeZone::detectHostTimeZone();
@@ -73,7 +73,7 @@
      static TimeZone* U_EXPORT2 createDefault(void);
  
 +#define ICU_TZ_HAS_RECREATE_DEFAULT
-+    static void U_EXPORT2 recreateDefault();
++    static void U_EXPORT recreateDefault();
 +
      /**
       * Sets the default time zone (i.e., what's returned by createDefault()) to be the
--- firefox-43.0b3/intl/icu/source/i18n/timezone.cpp.1	2015-11-19 12:38:43.902183332 +0300
+++ firefox-43.0b3/intl/icu/source/i18n/timezone.cpp	2015-11-19 12:39:13.559268961 +0300
@@ -569,7 +569,7 @@
     return (DEFAULT_ZONE != NULL) ? DEFAULT_ZONE->clone() : NULL;
 }
 
-void
+void U_EXPORT
 TimeZone::recreateDefault()
 {
     TimeZone *default_zone = TimeZone::detectHostTimeZone();
--- firefox-43.0b3/intl/icu/source/i18n/unicode/timezone.h.1	2015-11-19 12:39:37.086423989 +0300
+++ firefox-43.0b3/intl/icu/source/i18n/unicode/timezone.h	2015-11-19 12:39:49.677443243 +0300
@@ -305,7 +305,7 @@
     static TimeZone* U_EXPORT2 createDefault(void);
 
 #define ICU_TZ_HAS_RECREATE_DEFAULT
-    static void U_EXPORT2 recreateDefault();
+    static void U_EXPORT recreateDefault();
 
     /**
      * Sets the default time zone (i.e., what's returned by createDefault()) to be the
